cypress/integration/models/administrator/custom-migration-targets/custom-migration-target.ts

```diff
@@ -86,6 +130,30 @@ export class CustomMigrationTarget {
         }
     }
 
+    private fillRepositoryOriginForm(values: Partial<CMTRepositoryOrigin>) {
```

Comment:
fillRepositoryForm

Reply:
Solved in 2e54c55a1fa91b17b4fc06714ac0ca5e489f32b9.